# proj4 needs to set a global variable on the window
# object in order to use it as a module.
--- input
+++ output
@@ -2,7 +2,7 @@
     typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory() :
     typeof define === 'function' && define.amd ? define(factory) :
     (global.proj4 = factory());
-}(this, (function () { 'use strict';
+}(window, (function () { 'use strict';
 
     var globals = function(defs) {
       defs('EPSG:4326', "+title=WGS 84 (long/lat) +proj=longlat +ellps=WGS84 +datum=WGS84 +units=degrees");
